podium-gateway/src/main/webapp/app/backoffice/modules/organisation/organisation-form/organisation-form.component.html

```diff
@@ -83,11 +83,11 @@ <h2>
                                 </label>
                                 <input type="checkbox"
                                     [(ngModel)]="organisation.activated" name="organisation_active"
-                                    [disabled]="isOrganisationAdmin(currentAccount)" />
+                                    [disabled]="!canActivateOrganisation" />
 
-                                <div *ngIf="isOrganisationAdmin(currentAccount) && !organisation.activated">
+                                <div *ngIf="!canActivateOrganisation && !organisation.activated">
```

Comment:
This is a notice to the organisation admin, explaining that the activation needs to happen by the BBMRI admin, so it should be displayed to the organisation admin actually.

Reply:
If a user has both roles (BBMRI admin & Org admin), the original check shows an error message — that's definitely not what we want.

Reply:
Hm, looks like inactivated organisation not even show up for organisation admins. So we could remove this message altogether?

Reply:
That's right, an org admin can not see an inactive org in the list, but can still go to `bbmri/organisation/edit/{guid}` page directly. We can:
- disregard this case as a marginal one and remove excessive checks along with the message;
- make the page "forbidden" for an org admin when an org is inactive (and/or redirect somewhere else);
- keep the checks and the message in place, making sure it works correctly for any valid combination of roles for any organisation;

This PR implements the latter approach (as it requires the least amount of changes), but I can remove the checks & message if you think it would be better.

Reply:
We can disregard this case indeed. But I would remove the message, because it is not used/visible for anyone for whom it would be useful. Anyway, this message is not useful for the bbmri admin and should not be visible for the organisation admin. So, then better to remove it.